Automattic/woocommerce-payments
# Add Support for filtering by customer currency in analytics

Fixes #2535

#### Changes proposed in this Pull Request



- Adds a `Customer currency` filter to the Analytics page.







#### Testing instructions





* Make a few orders on your store in various different currencies, and wait a few minutes to ensure the action scheduler jobs which add the order info to the stats tables get processed
* Go to `Analytics > Orders` and verify you can see the `Customer currency` dropdown in the filters menu.
* Ensure you can filter by Customer Currency, and that selecting the `All options` option removes the filters. Make sure the total figures make sense (each value should reflect only the orders made in the selected currency).
* Quickly check the other analytics pages, they will **not** have this filter available, but just ensure the pages load and the data looks sensible.

Questions:

* This implements functionality that is (IMO) both usable and useful, but it doesn't implement *all* of the proposed changes in the original [project thread](https://wcpay.wordpress.com/2021/04/16/multi-currency-analytics/). I have split some of the additional functionality into follow-up issues. With that in mind, is it still okay to deploy this?

-------------------


**Post merge**
